syntax/clap_proj_tags.vim

```diff
@@ -9,7 +9,7 @@ syntax match ClapProjTagPattern /^.*$/ contains=ClapTagName,ClapProjTagKind,Clap
 hi default link ClapProjTagName Type
 hi default link ClapProjTagKind Function
 hi default link ClapProjTagPath Directory
-hi default link ClapProjTagPattern SpecialKey
+hi default link ClapProjTagPattern String
```

Comment:
I suggest using another one as `ClapProjTagKindPathSeperator` already uses String.

Reply:
Agree, changed to `Identifier`